docs/usage/saving.md

```diff
@@ -0,0 +1,56 @@
+# Saving
+
+To access the sidebar, click the hamburger icon (top left of window).
+
+![Sidebar image](../media/sidebar.png)
+
+General Info:
+
+- Choreo’s file saving mechanism ties into a WPILib robot root directory, wherein a single `.chor` file lives in the project’s root folder.
```

Comment:
IMO this would once again be better off as an info admonition than a single-item list.